portfolio/src/main/webapp/hobbies.html

```diff
@@ -10,5 +10,24 @@
   <body class="background">
     <h1 class="title">My Hobbies</h1>
     <div id="top-navigation"></div>
+     <table class="wics">
+      <tr>
```

Comment:
nit: I think these are only indented 1 space over. Please change to 2! Same with below!